index.template.html

```diff
@@ -5518,28 +5518,102 @@ <h1>JSON-LD Context Usage</h1>
+      <!-- 3 -->
+      <li>In section <a href="#terminology"></a>, a new WoT terminology <a>Canonical TD</a> was given definition.</li>
+      <!-- 4 -->
+      <li>In section <a href="#namespaces"></a>, Thing Model vocabulary was assigned a namespace IRI.</li>
+      <!-- 5.3.1.1 -->
+      <li>In section <a href="#thing"></a>:
+        <ul>
+          <li>New terms <code>base</code>, <code>securityDefinitions</code> and <code>profile</code> were added.</li>
+          <li>Type definitions for terms <code>version</code>, <code>support</code>, <code>security</code> were changed.</li> 
+        </ul>
+      </li>
+      <!-- 5.3.1.2 -->
+      <li>In section <a href="#interactionaffordance"></a>, the description for <code>uriVariables</code> was clarified.</li>
+      <!-- 5.3.1.3 -->
+      <li>In section <a href="#propertyaffordance"></a>, the description for <code>observable</code> was clarified.</li>      <!-- 5.3.1.6 -->
+      <li>In section <a href="#versioninfo"></a>, a new term <code>model</code> was added.</li>
+      <!-- 5.3.2.1 -->
+      <li>In section <a href="#dataschema"></a>:
+        <ul>
+          <li>Type definition for term <code>unit</code> was changed.</li>
+          <li>A new term <code>default</code> was added.</li>
+          <li>A note was added to inform readers of the status of <code>format</code> implementations and standardisation.</li>
+        </ul>
+      </li>
+      <!-- 5.3.2.4 -->
+      <li>In sections <a href="#numberschema"></a> and <a href="#integerschema"></a>, two new terms <code>exclusiveMinimum</code> and <code>exclusiveMaximum</code> were added.</li>
+      <!-- 5.3.2.7 -->
+      <li>In section <a href="#stringschema"></a>, a new term <code>pattern</code> was added.</li>
+      <!-- 5.3.3.1 -->
+      <li>In section <a href="#securityscheme"></a>:
+        <ul>
+          <li>It was noted that any private keys, passwords, or other sensitive information must not be stored in the TD.</li>
+          <li>Descriptions on the use of <code>in</code> were added, including how <code>in</code> together with <code>name</code> can be used to represent multiple parameters.</li>
+          <li>A string <code>uri</code> was added as one of valid values of <code>in</code>.</li>
+        </ul>
+      </li>
+      <!-- 5.3.3.6 -->
+      <li>In section <a href="#apikeysecurityscheme"></a>, the description of the schemes was improved.</li>
+      <!-- 5.3.3.8 -->
+      <li>In section <a href="#psksecurityscheme"></a>, the description of the schemes was improved.</li>
+      <!-- 5.3.4 -->
+      <li>In section <a href="#sec-hypermedia-vocabulary-definition"></a>:
+        <ul>
+          <!-- 5.3.4.1 -->
+          <li>In section <a href="#link"></a>:
+            <ul>
+              <li>A new terms <code>href</code> and <code>sizes</code> are now available.</li>
+              <li>Type definitions for terms <code>type</code>, <code>rel</code>, <code>anchor</code> were changed.</li>
```

Comment:
also a butg of the previews draft version. TD 1.0 used the right type

Reply:
sizes is new, but not href